src/WebJobs.Script/Binding/Http/RawScriptResult.cs

```diff
@@ -79,10 +79,12 @@ public async Task ExecuteResultAsync(ActionContext context)
                 {
                     if (cookie.Item3 != null)
                     {
+                        // CodeQL [SM02373] This code path constructs the cookie collection based on what the out-of-process function app (where customers can set these cookies) sends to the host. Marking these cookies as "Secure" would introduce a breaking change for those customers.
```

Comment:
We can probably replace `Marking these cookies as "Secure"` with `Overriding this behavior`

Reply:
Updated the comment as suggested.